core/server/helpers/t.js

```diff
@@ -0,0 +1,31 @@
+// # t helper
+// i18n: Translatable handlebars expressions for templates of the front-end and themes.
+// Front-end: .hbs templates in core/server, overridden by copies in themes. Themes: in content/themes.
+//
+// Usage examples, for example in .hbs theme templates:
+// {{t "Get the latest posts delivered right to your inbox"}}
+// {{{t "Proudly published with {ghostlink}" ghostlink="<a href=\"https://ghost.org\">Ghost</a>"}}}
+//
+// To preserve HTML, use {{{t}}}. This helper doesn't use a SafeString object which would prevent escaping,
+// because often other helpers need that (t) returns a string to be able to work as subexpression; e.g.:
+// {{tags prefix=(t " on ")}}
+
+var proxy = require('./proxy'),
+    jp = require('jsonpath'),
+    i18n = proxy.i18n,
+    settingsCache = proxy.settingsCache,
+    bindings = {},
+    path;
+
+module.exports = function t(text, options) {
+    var prop, theme;
+    for (prop in options.hash) {
+        if (options.hash.hasOwnProperty(prop)) {
+            bindings[prop] = options.hash[prop];
+        }
+    }
+    bindings.defaultString = text;
+    theme = settingsCache.get('active_theme') || 'casper';
+    path = jp.stringify(['$', 'theme', theme, text]);
```

Comment:
Why do we need a nested json structure in the theme locales?
That just makes it more complicated. I don't see the reasoning for 

A: why do we need a "theme" prefix?
B: why do we need to put the theme name?

Could you explain please? Thanks!

https://github.com/juan-g/Ghost/blob/6fcf7f460cea7c43ff05ece00fcd018a1f2042a3/content/locales/TRANSLATIONS.md#3-translate-the-included-sentences

Reply:
Those were needed to prevent conflicts in the old monolithic situation. As mentioned, with today's commit for separate `themeStrings` object in memory, etc., the JSON translation files for themes are much simpler now.